exposed-r2dbc/src/main/kotlin/org/jetbrains/exposed/v1/r2dbc/statements/api/R2dbcDatabaseMetadataImpl.kt

```diff
@@ -95,13 +95,15 @@ class R2dbcDatabaseMetadataImpl(
 
     override val identifierManager: IdentifierManagerApi by lazy {
         // db URL as KEY causes issues with multi-tenancy!
-        // TODO REVIEW use of JDBC url versus database here
+        // 'database' string may certainly be less complex a key than JDBC 'connection.url' value.
+        // To use an identical url string, we would need to save/parse it to/from ConnectionFactoryOptions & pass to this class.
+        // This is what is done for R2dbcDatabase.url, for example.
+        // So far this is the only use for us storing ConnectionFactoryOptions details in this class, but perhaps if other cases arise?
```

Comment:
we can try to expose the parse result in the public API from place available everywhere